src/Components/CollectionsHubRails/ArtistSeriesRail/index.tsx

```diff
@@ -14,40 +14,65 @@ export const ArtistSeriesRail: React.FC<ArtistSeriesRailProps> = ({
   collectionGroup,
 }) => {
   const { members } = collectionGroup
+  const broswerWidth = GetBroswerWidth()
```

Comment:
Since `GetBrowserWidth` isn't a component or a class, it should be camelCase: `getBrowserWidth`